cloud/deployment/deployment.go

```diff
@@ -1645,11 +1646,14 @@ func GetDeployment(ws, deploymentID, deploymentName string, disableCreateFlow bo
 	return currentDeployment, nil
 }
 
-func deploymentSelectionProcess(ws string, deployments []astroplatformcore.Deployment, deploymentFilter func(deployment astroplatformcore.Deployment) bool, platformCoreClient astroplatformcore.CoreClient, coreClient astrocore.CoreClient) (astroplatformcore.Deployment, error) {
+func deploymentSelectionProcess(ws string, deployments []astroplatformcore.Deployment, deploymentFilter func(deployment astroplatformcore.Deployment) bool, platformCoreClient astroplatformcore.CoreClient, coreClient astrocore.CoreClient, disableCreateFlow bool) (astroplatformcore.Deployment, error) {
 	// filter deployments
 	if deploymentFilter != nil {
 		deployments = util.Filter(deployments, deploymentFilter)
 	}
+	if len(deployments) == 0 && disableCreateFlow {
+		return astroplatformcore.Deployment{}, fmt.Errorf(errNoDeploymentInWSMsg, ws) //nolint:goerr113
```

Comment:
```suggestion
		return astroplatformcore.Deployment{}, fmt.Errorf("%w %s", ErrNoDeploymentInWS, ws)
```

Reply:
`//nolint:goerr113` would still be required since we are using dynamic error in this case, which the linter does not like 😞